### DISCUSSION TITLE
Plusieurs onglets dans S4E

### DISCUSSION THREAD

**Discussion by Tournesol01** [A-S-T-U-C-E/STudio4Education#54] [Ideas]
MIEUX... (mais plus ambitieux)

Si S4E permettait d'ouvrir 2 (ou plus) onglets ?

le «bag» deviendrait parfaitement inutile ⇒ un 2e onglet pourrait avantageusement servir de «bag» pour le premier ;
le passage d'un onglet à l'autre est bien plus rapide que le basculement entre 2 sessions, particulièrement si on travaille avec un seul écran : un seul clic suffit ;
ces onglets pourraient prendre place dans le bandeau blanc de STUDIO4EDUCATION, juste au dessus de la barre d'outils (comme dans beaucoup de logiciels : navigateurs par exemple, mais pas que...) ;
éventuellement, les onglets permettraient de travailler sur 2 ou 3 sketchs en parallèle.
(Par exemple, pour comparer rapidement quel est le meilleur de 2 sketchs pour obtenir l'effet voulu).
Et ceci d'autant plus que S4E intègre (en prévision) la totalité de la chaîne : blocs ⇒ code C++ ⇒ vérification du code ⇒ téléversement.
⇒ il devient alors très facile de téléverser alternativement un sketch puis l'autre, rien qu'en changeant d'onglet.
lors de la sauvegarde, tous les onglets seraient enregistrés en même temps (comme dans Excel par ex.), y compris donc le ou les onglets qui serviraient de «réserve de blocs».

**Comment by SebCanet** [comment]
A définir...ça reste en suspens, c'est un gros chantier mais là ce n'est pas une priorité.
